pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -489,6 +489,11 @@
             + " Using a value of 0, is disabling maxConsumersPerSubscription-limit check.")
     private int maxConsumersPerSubscription = 0;
 
+    @FieldContext(category = CATEGORY_SERVER, doc = "The size of messages.")
+    private int maxMessageSize = 5 * 1024 * 1024;
+
+    private int maxFrameSize = 5242880;
```

Comment:
Having 2 settings, which are ultimately strongly related, could be confusing to users. I think the max message size with some padding for headers would be easier in general.

Reply:
OK